### ISSUE TITLE
macOS Build doesn't support ch -? parameter

### ISSUE THREAD

**Issue by viccon** [Microsoft/ChakraCore#5625]
macOS Build doesn't support ch -? parameter

**Comment by rhuanjl** [comment]
It works for me on macOS - though you need to be using a test or debug build to get the full flag list; on a release build it gives a much smaller help text. I reccomend building a test build for most testing purposes (use the --test-build flag to the build.sh script)

**Comment by dilijev** [comment]
That's correct, -? displays the full list of test-flags on debug/test builds but only displays the same message as -h|-help on release builds, because there are no test-flags.
